CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+## How to contribute:
+
+It's recommended that you first contact ctsm-software@ucar.edu to discuss the changes or additions that you propose.
```

Comment:
I would suggest encouraging developers to open github issues instead of sending an email. Discussing new feature additions in the open will help provide transparency in the development process.

Reply:
Good point.